### ISSUE TITLE
Add 2 columns left sidebar layout

### ISSUE THREAD

**Issue by behindthepage** [backdrop/backdrop-issues#524]
I think having a layout for a 2 col with left sidebar as well as a 2 col with right sidebar in core is essential so I have created one. Watch this space.

**Comment by quicksketch** [comment]
I think the PR looks really good. We need to remove the settings.php changes of course, but otherwise it looks great.

I'd like to include this ASAP, but I think we should delay it until https://github.com/backdrop/backdrop-issues/issues/360 is completed, since we can always add another layout but we can only change the markup of our existing layouts before the initial release. One of the reasons we only have 3 layouts in the first place is because I wanted to get the markup/CSS worked out before we started maintaining a bunch of different layouts. Fortunately, that's all pretty well worked out, but we'll need to update this layout to match the new markup.

**Comment by behindthepage** [comment]
OK no problems.

**Comment by quicksketch** [comment]
Well that didn't take long with your review. We've got https://github.com/backdrop/backdrop-issues/issues/360 merged in now. If you can update the PR, I think this is a good idea for the usefulness of Backdrop.

**Comment by behindthepage** [comment]
Great. I will do a new PR so it is nice and clean and it is easier for me to do.

**Comment by quicksketch** [comment]
Fixed in https://github.com/backdrop/backdrop/pull/626. Thanks!
